packages/legend-extension-dsl-persistence/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_PersistenceTransformer.ts

```diff
@@ -0,0 +1,522 @@
+  context: V1_GraphTransformerContext,
+): V1_Persistence => {
+  const protocol = new V1_Persistence();
+  V1_initPackageableElement(protocol, element);
+  protocol.documentation = element.documentation;
+  protocol.trigger = V1_transformTrigger(element.trigger, context);
+  protocol.service = V1_transformElementReference(element.service);
+  protocol.persister = V1_transformPersister(element.persister, context);
+  protocol.notifier = V1_transformNotifier(element.notifier, context);
+  return protocol;
+};
+
+/**********
+ * trigger
+ **********/
+
+export const V1_transformTrigger = (
+  element: Trigger,
+  context: V1_GraphTransformerContext,
+): V1_Trigger => {
+  if (element instanceof CronTrigger) {
+    const protocol = new V1_CronTrigger();
+    protocol.minutes = element.minutes;
+    protocol.hours = element.hours;
+    protocol.dayOfMonth = element.dayOfMonth;
+    protocol.month = element.month;
+    protocol.dayOfWeek = element.dayOfWeek;
+    return protocol;
+  } else if (element instanceof ManualTrigger) {
+    return new V1_ManualTrigger();
+  }
+  throw new UnsupportedOperationError(
+    `Unable to transform trigger '${element}'`,
+  );
+};
+
+/**********
+ * persister
+ **********/
+
+export const V1_transformPersister = (
+  element: Persister,
+  context: V1_GraphTransformerContext,
+): V1_Persister => {
+  if (element instanceof StreamingPersister) {
+    const protocol = new V1_StreamingPersister();
+    if (element.connections) {
+      protocol.connections = element.connections.map((c) =>
+        V1_transformIdentifiedConnection(c, context),
+      );
+    }
+    return protocol;
+  } else if (element instanceof BatchPersister) {
+    const protocol = new V1_BatchPersister();
+    if (element.connections) {
+      protocol.connections = element.connections.map((c) =>
+        V1_transformIdentifiedConnection(c, context),
+      );
+    }
+    protocol.targetShape = V1_transformTargetShape(
+      element.targetShape,
+      context,
+    );
+    return protocol;
+  }
+  throw new UnsupportedOperationError(
+    `Unable to transform persister '${element}'`,
+  );
+};
+
+/**********
+ * notifier
+ **********/
+
+export const V1_transformNotifier = (
+  element: Notifier,
+  context: V1_GraphTransformerContext,
+): V1_Notifier => {
+  const notifier = new V1_Notifier();
+  notifier.notifyees = element.notifyees.map((n) =>
+    V1_transformNotifyee(n, context),
+  );
+  return notifier;
+};
+
+export const V1_transformNotifyee = (
+  element: Notifyee,
+  context: V1_GraphTransformerContext,
+): V1_Notifyee => {
+  if (element instanceof EmailNotifyee) {
+    const protocol = new V1_EmailNotifyee();
+    protocol.address = element.address;
+    return protocol;
+  } else if (element instanceof PagerDutyNotifyee) {
+    const protocol = new V1_PagerDutyNotifyee();
+    protocol.url = element.url;
+    return protocol;
+  }
+  throw new UnsupportedOperationError(
+    `Unable to transform notifyee '${element}'`,
```

Comment:
Let's change all message format from `Unable` -> `Can't` to be consistent with the codebase.

Reply:
Fixed in #990